docker-compose.yml

```
@@ -77,25 +91,52 @@ services:
+      config:
+        - subnet: 172.20.0.0/16
 
 volumes:
   postgres_data:
+    driver: local
   redis_data:
-  qdrant_data:
\ No newline at end of file
+    driver: local
+  qdrant_data:
+    driver: local
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)